### ISSUE TITLE
galaxy runner bug when testing linchpin develop with carbon

### ISSUE THREAD

**Issue by samvarankashyap** [CentOS-PaaS-SIG/linchpin#1439]
**Describe the bug**
Reported by @Dannyb48 
The following errors were observed when linchpin is tested against 
```
Traceback (most recent call last):
  File "/home/jenkins/workspace/e2e-openstack-linchpin-2/cbn/lib/python2.7/site-packages/blaster/engine/parallel.py", line 45, in run
    task_obj = task_cls(**task_def)
  File "/home/jenkins/workspace/e2e-openstack-linchpin-2/cbn/lib/python2.7/site-packages/carbon/tasks/provision.py", line 53, in __init__
    plugin = getattr(asset, 'provisioner_plugin')(asset)
  File "/home/jenkins/workspace/e2e-openstack-linchpin-2/cbn/lib/python2.7/site-packages/carbon/provisioners/ext/linchpin_wrapper_plugin.py", line 72, in __init__
    self.linchpin_api = LinchpinAPI(self._init_context())
  File "/home/jenkins/workspace/e2e-openstack-linchpin-2/cbn/lib/python2.7/site-packages/linchpin/__init__.py", line 92, in __init__
    for path in galaxy_runner.get_role_paths():
  File "/home/jenkins/workspace/e2e-openstack-linchpin-2/cbn/lib/python2.7/site-packages/linchpin/galaxy_runner.py", line 41, in get_role_paths
    end_idx = line.index(after)
ValueError: substring not found
```

**Comment by samvarankashyap** [comment]
@Dannyb48 Feel free to add additional details like the scenario file being used etc., in carbon.

**Comment by samvarankashyap** [comment]
#1438  fixes this issue
@Dannyb48

**Comment by samvarankashyap** [comment]
#1438 didn't actually fix this issue

**Comment by 14rcole** [comment]
Fixed by #1443
